static/app/utils/string/capitalize.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import {capitalize} from 'sentry/utils/string/capitalize';
+
+describe('capitalize', () => {
+  it('capitalizes the first letter of a string', () => {
```

Comment:
Capitalize also lowercases the rest of the string. Can you add a test case?

https://lodash.com/docs/4.17.15#capitalize